ops/internal/manage/collect_test.go

```diff
@@ -24,13 +24,13 @@ func TestCollectChainConfigs(t *testing.T) {
 		{
 			ShortName:  "op",
 			Filepath:   paths.ChainConfig("testdata", "sepolia", "op"),
-			Superchain: config.SepoliaSuperchain,
+			Superchain: "sepolia",
```

Comment:
Why did you change this from the enum to strings?

Reply:
See other comment. This change can be reverted after rebasing on top of `import_devnet` pr